notebooks/example.ipynb

```diff
@@ -13,7 +13,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 45,
```

Comment:
I think we should stick with having this be a fresh run each time it is committed.

Reply:
Instead of updating these each time or changing them if examples are reordered, I think it may be best to delete all of the outputs of the cells. I think Paul recommended this earlier as a best practice as well. Are you fine with me making that change instead?

Reply:
I think it might be the case that, if we delete the output cells, the nbviewer preview in binder may not have anything to show. That might be worth a test. Otherwise, I think clearing the cells and doing a _Run all cells_ might keep the numbers constant across commits and minimize the diffs.

Reply:
Correct, it does look like it removes the nbviewer preview. I will Run All Cells, that should standardize the numbers in order.